### PR TITLE
pysync: replace md5 by sha256

### PR DESCRIPTION
**Pull request by slari-pivotal**
This utility used to confirm data transfered by doing a md5 digest. This
commit changes behavior to use sha256 instead.

Signed-off-by: Nadeem Ghani <nghani@pivotal.io>

### CODE REVIEW

**Comment by chrishajas** [comment]
It would be helpful to split the whitespace changes into a separate commit for reviewing/future git blame purposes.

File: gpMgmt/bin/gppylib/test/unit/test_unit_pysync.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import os, tempfile, hashlib
+from gp_unittest import *
+from lib.pysync import LocalPysync
+
+CONTENT = 'To be or not to be.'
```
Comment (Chibin) [inline gpMgmt/bin/gppylib/test/unit/test_unit_pysync.py]: 👍

File: gpMgmt/bin/lib/pysync.py [hunk 2]
```diff
@@ -351,32 +357,33 @@ def __init__(self, argv, recordProgressCallback=None, recordRawProgressCallback=
                 except ValueError:
                     raise ValueError("--progress-time value is not supported", a[0])
                 if progressSeconds < pysync_remote.SyncProgress.MINIMUM_TIME_INTERVAL:
-                    raise ValueError("--progress-time value must be at least %f" % (pysync_remote.SyncProgress.MINIMUM_TIME_INTERVAL / 60))
-            
+                    raise ValueError("--progress-time value must be at least %f" % (
+                    pysync_remote.SyncProgress.MINIMUM_TIME_INTERVAL / 60))
```
Comment (Chibin) [inline gpMgmt/bin/lib/pysync.py]: Surprised it got reformatted like this instead of tabbed in once.

Comment (slari-pivotal) [reply gpMgmt/bin/lib/pysync.py]: IDE made this change when we reformatted.  We have manually fixed it now.

**Comment by slari-pivotal** [comment]
@chrishajas, we have split the commit into whitespace commit and the code changes.

**Pull request merged**